web/app/SuoritetutTutkinnot.tsx

```diff
@@ -0,0 +1,103 @@
+import { isSuccess, useApiWithParams } from './api-fetch'
+import { fetchSuoritetutTutkinnot } from './util/koskiApi'
+import * as R from 'ramda'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { isSuoritetutTutkinnotOppija } from './types/fi/oph/koski/suoritusjako/suoritetuttutkinnot/SuoritetutTutkinnotOppija'
+import { SuoritusjakoTopBar } from './components-v2/layout/SuoritusjakoTopBar'
+import { t } from './i18n/i18n'
+import { ContentContainer } from './components-v2/containers/ContentContainer'
+import { PlainList } from './components-v2/containers/PlainList'
+import { isSuoritetutTutkinnotAmmatillinenOpiskeluoikeus } from './types/fi/oph/koski/suoritusjako/suoritetuttutkinnot/SuoritetutTutkinnotAmmatillinenOpiskeluoikeus'
+import { Trans } from './components-v2/texts/Trans'
+import { Flex } from './components-v2/containers/Flex'
+// @ts-ignore
+__webpack_nonce__ = window.nonce
+// @ts-ignore
+import(/* webpackChunkName: "styles" */ './style/main.less')
+
+const secret = R.last(document.location.pathname.split('/')) ?? ''
+interface SuoritetutTutkinnotProps {
+  testId?: string
+}
+const SuoritetutTutkinnot = ({ testId }: SuoritetutTutkinnotProps) => {
+  const response = useApiWithParams(fetchSuoritetutTutkinnot, [secret])
+  return isSuccess(response) && isSuoritetutTutkinnotOppija(response.data) ? (
+    <>
+      <SuoritusjakoTopBar />
+      <ContentContainer
+        className="content-area suoritusjako suoritusjako-page"
+        testId={testId}
+      >
+        <div className="main-content">
+          <h1>
+            <Trans>{'Suoritetut tutkinnot'}</Trans>
+          </h1>
+          <a
+            className="text-button-small"
+            target="_blank"
+            href={`/koski/api/opinnot/suoritetut-tutkinnot/${secret}`}
+            rel="noopener noreferrer"
+          >
+            <Trans>{'Tiedot koneluettavassa muodossa'}</Trans>
+          </a>
+          <div>
+            <div className="header__name">
+              <p className="textstyle-like-h2">
+                {response.data.henkilö.etunimet}{' '}
+                {response.data.henkilö.sukunimi}
+              </p>
+            </div>
+          </div>
+
+          <div></div>
+
+          <div>
+            <p>
+              <Trans>
+                {
+                  'Tässä listataan suoritetut tutkinnot otsikkotasolla. Tarkemmat tiedot löytyvät koneluettavassa muodossa yllä olevasta linkistä.'
```

Comment:
Uudet kälimerkkijonot pitäisi vielä lisätä koski-default-texts.json :iin, jotta päätyvät myös lokalisointipalveluun.

Tämä on vähän epäkäytännöllistä, mutta näin tämä Koskessa toimii, eikä kälimerkkijonojen lisäämiselle ole oikein mitään automaattitestiäkään, joka huomaisi, että puuttuvat...